lib/routes/token-codes.js

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const errors = require('../error')
+const isA = require('joi')
+const validators = require('./validators')
+const HEX_STRING = validators.HEX_STRING
+
+module.exports = (log, db, customs) => {
+  return [
+    {
+      method: 'POST',
+      path: '/tokenCodes/verify',
```

Comment:
As an additional layer of protection against brute-forcing attacks, would it be worth having this endpoint be authenticated with the corresponding `sessionToken`?

Reply:
And if we *do* make this be authenticated with the session token, I think the API surface would be a lot more manageable if we exposed it as `/session/verify_code (🔒 sessionToken)` or similar, by analogy to the API namespace under `/password/forgot`.  But I'll try to resist the urge to bikeshed this too much! :grin:

Reply:
Making a more explicitly-named endpoint under `/session` sounds good to me.  We could do something like `/session/verify_token_code` and `/session/verify_hotp_code` for example, or `/session/verify/(token|hotp|whatever)`.  I don't have a strong preference on the details there.